hexrd/ui/image_canvas.py

```diff
@@ -233,6 +248,28 @@ def draw_detector_borders(self):
 
         self.draw()
 
+    def reformat_overlays(self, data):
+        x, y = data[:, 1], data[:, 0]
+        if self.mode == 'cartesian':
+            # It's centered on (0, 0). Recenter it in place.
+            self.recenter_points((0, 0), x, y)
+        elif self.mode == 'polar':
+            # Swap axes and convert to degrees.
+            x, y = np.degrees(y), np.degrees(x)
+            # It's centered on 180. Recenter on 0.
+            y -= 180.0
+
+        return x, y
```

Comment:
Is this necessary?

Reply:
I do need to convert it to degrees and swap the axes, but maybe I can do those in the powder overlay class instead of here.

Without the re-centering on 0, it looks like this:

![polar](https://user-images.githubusercontent.com/9558430/84192986-c6643c80-aa68-11ea-9209-7aabb261e7bb.png)

But actually, since we are drawing vertical lines over the whole image, I can probably just use [axvline](https://matplotlib.org/3.1.0/api/_as_gen/matplotlib.axes.Axes.axvline.html) in matplotlib. That way, I don't need to supply y coordinates, and it won't be an issue.

Reply:
I have moved the reformatting of the polar view to the overlay class in b75d04645ccd60bda2b7523c12d4169b74fdf8fd. This should hopefully be resolved.